### DISCUSSION TITLE
badge for profile github

### DISCUSSION THREAD

**Discussion by subastianSk** [community/community#27786] [Other Feature Feedback, Questions, & Ideas]
how do you get an award on your profile?

**Comment by MrSarno** [comment]
Hey @subastianSk,

There's a list of achievements & notes on how to unlock them [here](https://github.com/Schweinepriester/github-profile-achievements).

If this helps, please mark it as the answer so that the question no longer appears in [the list](https://github.com/orgs/community/discussions?discussions_q=is%3Aunanswered) of unanswered questions. Thanks! 👍
